kolibri/tasks/api.py

```diff
@@ -191,7 +280,7 @@ def localdrive(self, request):
         return Response(out)
 
 
-def _networkimport(channel_id, update_progress=None, check_for_cancel=None):
+def _networkimport(channel_id, node_ids, update_progress=None, check_for_cancel=None):
```

Comment:
Do we still need `_networkimport`, `_localimport`, `_localexport`, `_deletechannel` functions?

Reply:
Not anymore -- i kept it in so we can reference it until it's fully implemented.